src/hooks/useSortAndSearch.jsx

```diff
@@ -0,0 +1,21 @@
+import { useMemo } from 'react';
+
+export const useSortAndSearch = (todos, sort, query) => {
+  const modifiedTodos = useMemo(() => {
+    return (todos
+      .filter(todo => {
+        switch (sort) {
+          case 'active':
```

Comment:
Better create Enum with all possible sorts in order not to hardcore them and make your code cleaner